src/Components/SurveyComponent.js

```diff
@@ -65,20 +69,23 @@ const Outcome = ({ type }) => {
                 <div>
                     <h2>You are eligible!</h2>
                     <div>blah blah blah</div>
+                    <DemographicSurveyComponent surveyModel={demoSurveyModel} />
                 </div>
             );
         case 'needInfo':
             return (
                 <div>
                     <h2>Need more info</h2>
                     <div>go here ask this ....</div>
+                    <DemographicSurveyComponent surveyModel={demoSurveyModel} />
                 </div>
             );
         case 'ineligible':
             return (
                 <div>
                     <h2>Sorry not eligible</h2>
                     <div>here are resources</div>
+                    <DemographicSurveyComponent surveyModel={demoSurveyModel} />
                 </div>
             );
         default:
```

Comment:
I was thinking maybe we could refactor `Outcome` to reference a contact map for the unique content, and then any shared content (like `DemographicSurveyComponent`) would only need to be included once. What do you think of something like this?

```js
const outcomeMap = {
    eligible: (
        <div>
            <h2>You are eligible!</h2>
            <div>blah blah blah</div>
        </div>
    ),
    needInfo: (
        <div>
            <h2>Need more info</h2>
            <div>go here ask this ....</div>
        </div>
    ),
    ineligible: (
        <div>
            <h2>Sorry not eligible</h2>
            <div>here are resources</div>
        </div>
    ),
};

const Outcome = ({ type }) => {
    return (
        <div>
            {outcomeMap[type]}
            <DemographicSurveyComponent surveyModel={demoSurveyModel} />
        </div>
    );
};
```

Eventually it probably makes sense to move `Outcome` to its own file, since it will grow as we fill in the real content